packages/components/src/card/style.scss

```diff
@@ -3,19 +3,35 @@
 .woocommerce-card {
 	margin-bottom: $gap-large;
 	background: $studio-white;
-	border-radius: 3px;
-	box-shadow: $muriel-box-shadow-1dp;
-	transition: box-shadow 0.2s cubic-bezier(0.4, 1, 0.4, 1);
+	border-radius: 0;
```

Comment:
These style changes will break card styling in onboarding.

Reply:
Thanks @joshuatf, I was fearing this. And sorry I didn't see your open PR.

>Are we using the default Card styling anywhere? If not, we might want to consider making the default style the same as what we're using in analytics.

~ from @jeffstieler https://github.com/woocommerce/woocommerce-admin/pull/3267#issuecomment-555016688

The answer I found is no. So how about adding an `onboarding_card` (or something) class to onboarding's use of Card to keep it separate? That will override base styles. One reason to do this the exported `Card` styles are broken and would remain so in your PR.